.github/scripts/backport/main.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+GITHUB_TOKEN=$GITHUB_TOKEN
+
+git config --global user.email "noreply@github.com"
+git config --global user.name "GitHub Actions"
+
+git config --global credential.helper "store --file=~/.git-credentials"
+git config --global credential.https://github.com.username $GITHUB_TOKEN
+
+echo "Successfully authenticated!"
+
+git checkout $GITHUB_EVENT_BRANCH
+echo "Checked out main"
+
+# Get backport label from all the labels of the PR
+labels=$(jq --raw-output '.pull_request.labels[].name' "$GITHUB_EVENT_PATH")
+backport_label=""
+branch_name=""
+for label in "${labels[@]}"; do
+    echo "each label - $label"
+    if [[ $label == "backport"* ]]; then
+        backport_label=$label
+        branch_name=${label#backport }
+        break
+    fi
+done
+echo "Label Name: $backport_label"
+
+# Checkout the version branch
+git fetch --all
+git checkout $branch_name
+echo "Version branch: $branch_name"
+
+# Checkout new backport branch
+ISSUE_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
```

Comment:
Let's do all of the issue parsing in the same location in the script for clarity.  Also, I'd prefer to do as much of this as possible in the github workflow file and not in a bash script.

Reply:
Also why is this uppercase when all the rest of the vars are lower case?

Reply:
I tested the commands on my terminal on personal repo and didn't notice the variable naming consistency.